xarray/core/dataarray.py

```diff
@@ -1461,7 +1461,7 @@ def transpose(self, *dims, transpose_coords=None) -> 'DataArray':
     def T(self) -> 'DataArray':
         return self.transpose()
 
-    def drop(self, labels, dim=None):
+    def drop(self, labels, dim=None, errors='raise'):
```

Comment:
Let's make this new argument require using a keyword argument: 
```suggestion
    def drop(self, labels, dim=None, *, errors='raise'):
```

Reply:
Yes, let's make that keyword argument only, too.